weave-js/src/components/PagePanelComponents/Home/Browse3/pages/PlaygroundPage/PlaygroundPage.tsx

```diff
@@ -32,15 +37,25 @@ type PlaygroundPageInnerProps = PlaygroundPageProps &
     setSettingsTab: Dispatch<SetStateAction<number | null>>;
   };
 
+const isLLMMaxTokensKey = (key: string): key is LLMMaxTokensKey => {
+  return key in LLM_MAX_TOKENS;
+};
+
 export const PlaygroundPage = (props: PlaygroundPageProps) => {
   const [settingsTab, setSettingsTab] = useState<number | null>(null);
 
+  // We expect the passed in models to all be in LLM_MAX_TOKENS,
```

Comment:
just from skimming the variable names, this seems weird. Why would `initialModelStrings` be in `LLM_MAX_TOKENS`?

Reply:
Despite the name, LLM_MAX_TOKENS is used for more than that, like constructing the model select UI. Agree it is a confusing name but was trying to minimize changes to existing code in already overly big PR.